semantics/src/traversal/functions.rs

```diff
@@ -106,14 +106,28 @@ fn func_stmt(
         fe::FuncStmt::While { .. } => unimplemented!(),
         fe::FuncStmt::If { .. } => unimplemented!(),
         fe::FuncStmt::Assert { .. } => unimplemented!(),
-        fe::FuncStmt::Expr { .. } => unimplemented!(),
+        fe::FuncStmt::Expr { .. } => expr(scope, context, stmt),
```

Comment:
I could be wrong, but I think this variant would only be used for function call statements.

If I were working on this, I'd probably start by directing this straight into a method named something like `func_stmt_call` that is pretty similar to the [`expr_call`](https://github.com/ethereum/fe/blob/de661d0e8d81d006b09319413b9fab5cff93c956/semantics/src/traversal/expressions.rs#L247) method. The only difference would be that we don't return a semantic error for empty return calls and that we don't create expression attributes for the call statement.

Reply:
Wouldn't we also hit this for any other expression such as `1+1` in the following example.

```
    pub def bar() -> u256:
        1 + 1
        return 100
```

Obviously, the `1 + 1` doesn't really do anything and an optimizer would probably optimize it away but I assume it is still valid.

Reply:
I'm ok with allowing this sort of thing now.